XenAdmin/TabPages/PhysicalStoragePage.cs

```diff
@@ -328,74 +282,59 @@ private void newSRButton_Click(object sender, EventArgs e)
             new NewSRCommand(Program.MainWindow, connection).Execute();
         }
 
-        /// <summary>
-        /// Taken from http://support.microsoft.com/kb/319401.
-        /// </summary>
-        /// <param name="sender"></param>
-        /// <param name="e"></param>
-        private void listViewSrs_ColumnClick(object sender, ColumnClickEventArgs e)
+        private void dataGridViewSr_MouseUp(object sender, MouseEventArgs e)

```

Comment:
I believe the `newSRButton_Click(object sender, EventArgs e)` method above is not used, hence can be removed.